widgets/contact/fields/radio.class.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class SiteOrigin_Widget_ContactForm_Field_Radio extends SiteOrigin_Widget_ContactForm_Field_Base {
+
+	public function render_field( $options ) {
+
+		if ( ! empty( $options['field']['options'] ) ) {
+			?><ul><?php
```

Comment:
This just needs a small style improvement. It should be using the [alternative control structure](http://php.net/manual/en/control-structures.alternative-syntax.php) because it's a mix of HTML and PHP. This makes the code look a lot neater and you can keep the format more HTML friendly.